src/utilities/MakerUtility.sol

```diff
@@ -8,6 +8,8 @@ import {IDSProxy, IDSProxyRegistry} from '../interfaces/maker/IDSProxy.sol';
 import {IMakerGemJoin} from '../interfaces/maker/IMaker.sol';
 import {ApproveHelper} from '../libraries/ApproveHelper.sol';
 
+/// @title Maker utility contract
+/// @notice Perform additional actions after interacting with Maker
```

Comment:
Should be before? I thought user will call utility contract instead of calling Maker.

Reply:
This comment indicates that the transfers of DAI and CDP occur after interacting with Maker in each functions in Utility, However, I agree it might be confusing.

I suggest rewording it to:  Perform additional actions `when` interacting with Maker